1732Sprint/3-Sheik/Chapter5/classes/Character.java

```diff
@@ -0,0 +1,22 @@
+package classes;
+
+public abstract class Character {
+
+    public String name;
```

Comment:
You sure this should be public? I see that I can create a Knight character and then give him the name, 'Bird', no?